src/CMakeLists.txt

```diff
@@ -280,10 +280,8 @@ add_subdirectory(box)
 include_directories(${EXTRA_BOX_INCLUDE_DIRS})
 
 # Save CMAKE_XXX_FLAGS from this directory for config.h (used in --version)
-string(REPLACE  ${PREFIX_MAP_FLAGS} "" STRIPPED_CMAKE_C_FLAGS ${CMAKE_C_FLAGS})
-set(TARANTOOL_C_FLAGS ${STRIPPED_CMAKE_C_FLAGS} PARENT_SCOPE)
-string(REGEX REPLACE ${PREFIX_MAP_FLAGS} "" STRIPPED_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
-set(TARANTOOL_CXX_FLAGS ${STRIPPED_CMAKE_CXX_FLAGS} PARENT_SCOPE)
```

Comment:
`PREFIX_MAP_FLAGS` is still there (and contains `-fmacro-prefix-map=<source dir>=.`) and, to be honest, I don't understand two things:

* Why we strip it in the first place?
* Why now we don't do that?

This point looks minor, so feel free to ignore.

Reply:
Okay.

I would like to have details that are not obvious from the code itself in a commit message, but I don't insist -- closing the discussion so.

Reply:
> I would like to have details that are not obvious from the code itself in a commit message.

I will keep that in mind for the future, thanks for your feedback!